### PR TITLE
[Docs] update data types doc and fix some typo

### LINKED ISSUE #4633: HLL文档描述错误
http://doris.apache.org/master/zh-CN/sql-reference/sql-statements/Data%20Types/HLL.html
description里描述的是`VARCHAR`

### LINKED ISSUE #4634: SQL手册中缺少BITMAP类型介绍
http://doris.apache.org/master/zh-CN/sql-reference/sql-statements/Data%20Types/BIGINT.html
缺少BITMAP类型

### LINKED ISSUE #4635: VARCHAR文档错误
[文档](http://doris.apache.org/master/zh-CN/sql-reference/sql-statements/Data%20Types/VARCHAR.html#description)介绍VARCHAR最大长度为65535，但实际应为65533
```
mysql(sduzh)> create table t0(k varchar(65535)) distributed by hash(k) buckets 1;
ERROR 1064 (HY000): errCode = 2, detailMessage = Varchar size must be <= 65533: 65535
```

### PR DESCRIPTION
**Pull request by Astralidea**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Fix #4633 
Fix #4634 
Fix #4635 

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/sql-reference/sql-statements/Data Types/BITMAP.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# BITMAP
+## description
+    BITMAP
+    BITMAP不能作为key列使用，建表时配合聚合类型为BITMAP_UNION。
+    用户不需要指定长度和默认值。长度根据数据的聚合程度系统内控制。
+    并且BITMAP列只能通过配套的bitmap_union_count、bitmap_union、bitmap_hash等函数进行查询或使用。
+    
+    离线场景下使用BITMAP会影响导入速度，在数据量大的情况下查询速度会慢于HLL，并优于Count Distinct。
+    实时场景下BITMAP也不能做到完全的精确去重，通常误差小于千分之一。
```
Comment (morningman) [inline docs/zh-CN/sql-reference/sql-statements/Data Types/BITMAP.md:35]: 为什么bitmap在实时场景下不能做到精确去重呢？

Comment (Astralidea) [reply docs/zh-CN/sql-reference/sql-statements/Data Types/BITMAP.md:35]: 实时场景下的BITMAP，通常不会引入全局字典，因为当全局字典大了之后时延很大，实时的吞吐和时延都会明显下降。导致查询和更新全局字典的成本和时延很高，会极大降低实时导入的性能。
这块文档需要特别说明一下吗？

Comment (morningman) [reply docs/zh-CN/sql-reference/sql-statements/Data Types/BITMAP.md:35]: 哦，这个描述不太准确。bitmap这个类型本身是精确的。只是在“实时场景”下，用户导入的数据本身是不精确的。比如不使用全局字典，或者使用了 bitmap_hash() 等函数导致的误差，这些和bitmap本身的实现无关。这个需要说明下。

**Comment by Astralidea** [comment]
这个仓库又让我搞没了，换到另一个 #4712 吧，我改了一下描述，看看还有啥问题。

**Pull request closed without merging**